site/content/docs/user/ingress.md

```diff
@@ -156,7 +156,7 @@ spec:
 Apply the contents
 
 ```shell script
-kubectl apply -f https://kind.sigs.k8s.io/manifests/ingress/nginx/example.yaml
+kubectl apply -f {{< baseurl >}}manifests/ingress/nginx/example.yaml
```

Comment:
it's missing one `/` ???

Reply:
it's an inconsistency between local dev and netlify build

the correct answer is using the `absURL` method, which I've updated to.

Reply:
locally there's a trailing slash, in CI there is not. absURL handles this appropriately.